elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -93,18 +93,20 @@ class PfeJumpLinksNav extends PFElement {
         if (this.getAttribute("sr-text")) {
           this.shadowRoot.querySelector("nav").prepend(div);
         }
-
+        if (this.hasAttribute("horizontal")) {
+          return;
+        }
         let html = "";
-        if (this.querySelector(".pfe-jump-links-nav--heading")) {
-          html = this.querySelector(".pfe-jump-links-nav--heading").cloneNode(
-            true
-          );
+        if ("[slot='pfe-jump-links-nav--heading']") {
```

Comment:
```suggestion
        if (this.has_slot("pfe-jump-links-nav--heading")) {
```

There's a built-in function in pfelement that can verify if a slot exists.  Alternatively, I think this needs a query selector instead of just the quotes.

Reply:
You're right, good catch. I wonder why this worked though? Weird.

Reply:
I suspect it's always true.  Maybe we need to add an AT that validates that the default text is used if the heading slot is empty?